dali/pipeline/data/types.h

```diff
@@ -636,6 +636,12 @@ inline std::ostream &operator<<(std::ostream &os, DALIDataType dtype) {
         {__VA_ARGS__}                                \
       }                                              \
       break;                                         \
+    case DALI_INT8:                                  \
```

Comment:
Food for thought:

This macro is used only once - I'd remove it, add some definition of tensor-compatible types and use `TYPE_SWITCH` instead.
```C++
#define DALI_INTEGRAL_TYPES uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t
#define DALI_NUMERIC_TYPES DALI_INTEGRAL_TYPES, float, double
#define DALI_NUMERIC_TYPES_FP16 DALI_NUMERIC_TYPES, float16
```

Reply:
Done